### DISCUSSION TITLE
When will chia solve the endless BUG?

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain#11438] [Support]
When will chia solve the endless BUG?

**Comment by loppefaaret** [comment]
would you be able to give us a discription of which bug you are encoutering ?

**Comment by NTOHPN** [comment]
1.plots check is not available now. I don't know if it's because of the python version.
two。. The harvester cannot use more than 50 hard drives on a single computer, and if it exceeds it, the output calculation is not normal.
3. The GUI that exceeds the capacity of 1PB runs slowly and the interface is stuck.

**Reply by loppefaaret** [reply]
lets stick to #11436
